lib/lib.go

```diff
@@ -184,14 +184,11 @@ func OptCheckConfigMigrations(cfgPath string) Option {
 			return fmt.Errorf("no config file to check for migrations")
 		}
 
-		migrated, err := migrate.RunMigrations(o.Streams, o.Cfg)
+		_, err := migrate.RunMigrations(o.Streams, o.Cfg)
```

Comment:
Remove the first return value if it's not needed. We can tell what happened from the `err` alone.